examples/megatron/models/vllm_policy_inference.py

```diff
@@ -66,7 +71,7 @@ def eval_forward(self, data, iteration=0):
 
     def _forward_step(self, data, iteration, is_eval): # pylint: disable=unused-argument
         outputs = self.generate_vllm(data, is_eval)
-        if self.is_last_rank():
```

Comment:
why delete self.is_last_rank()

Reply:
for VLLMMoudleV2, the output is from engine, so there is one output for one replica, which is different from previous settings (last rank got output, other rank got None)

Reply:
How to compatible with VLLMModule v1

Reply:
it is compatible now, we use `if outputs is not None` instead